packages/website/src/docs/toggle-button.mdx

```diff
@@ -39,6 +39,11 @@ description: Useful for allowing users to select one or more options from a set.
       type: 'ReactNode',
       description: 'The content to render',
     },
+    variant: {
+      type: 'normal | fade',
+      default: 'normal',
+      description: 'The variant to apply',
+    },
     size: {
       type: '0 | 1',
       default: '0',
```

Comment:
unrelated but shouldn't the default be `1`  here?

Reply:
It should! df1ff999877ffb094ff7c28314e8d24c435471eb